packages/urls/src/urls.ts

```diff
@@ -15,6 +15,7 @@ export const SUITE_BACKUP_URL = 'https://suite.trezor.io/web/backup/';
 export const SUITE_BRIDGE_URL = 'https://suite.trezor.io/web/bridge/';
 export const SUITE_FIRMWARE_URL = 'https://suite.trezor.io/web/firmware/';
 export const SUITE_UDEV_URL = 'https://suite.trezor.io/web/udev/';
+export const SUITE_BRIDGE_DEEPLINK = 'trezorsuite://bridge-requested-by-a-3rd-party';
```

Comment:
this list was intended for urls accessible over the internet. there is a nightly check that those links return some non 4** response

Reply:
Where should we put it? Maybe `src/constants/suite/protocol'`?

Reply:
can every part that needs it import from there? If yes, it sounds good to me

Reply:
Oh, probably not, we'll need it in Connect. So maybe a new file in urls package for deeplinks?

Reply:
good choice!